spec-0007/index.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "SPEC 7 — Seeding pseudo-random number generation"
+date: 2023-04-19
+author:
+  - "Stéfan van der Walt <stefanv@berkeley.edu>"
+  - "Sebastian Berg <sebastianb@nvidia.com>"
+  - "Pamphile Roy <roy.pamphile@gmail.com>"
+  - "Matt Haberland <mhaberla@calpoly.edu>"
+  - Other participants in the discussion <not.yet@named.org>"
+discussion: https://github.com/scipy/scipy/issues/14322
+endorsed-by:
+---
+
+## Description
+
+Currently, libraries across the ecosystem provide various APIs for seeding random number generation.
+This SPEC suggests a unified, pragmatic API, taking into account technical and historical factors.
+Adopting such a uniform API will simplify the user experience, especially for those who rely on multiple projects.
+
+We recommend:
+
+- avoiding the use of global state and legacy bitstream generators, and
+- standardizing the usage and interpretation of an `rng` keyword for seeding.
+
+We suggest implementing these principles by:
+
+- deprecating the use of `numpy.random.seed` to control the random state,
```

Comment:
Maybe should clarify that this recommendation is for downstream libraries, not NumPy itself.
And I suppose in the end we didn't exactly *deprecate* this either : / But I suppose I'm splitting hairs about the word.